### ISSUE TITLE
the warning and strange number of flops

### ISSUE THREAD

**Issue by BitCalSaul** [zugexiaodui/torch_flops#3]
Thanks for your work and I want to implement it to see my new model's performance. This is the first tool I have seen so far that could count some special operation in the forward process but not limited in just nn.Module.
I installed this tool and tried to count a simple linear model's flops. But the results seem strange with some warning. Would you please check what's going on? Thanks.

This is code.
```
import torch
import torch.nn as nn
from torch_flops import TorchFLOPsByFX

device = torch.device('cuda:3')

model = WindowAttention(dim=dim, window_size=(8, 8), num_heads=1).to(device)

dim = 192

model = nn.Linear(dim, dim*3)

x = torch.randn([1, 64, 64, dim])

print("=" * 10, "model", "=" * 10)


flops_counter = TorchFLOPsByFX(model)

flops_counter.propagate(x)

total_flops = flops_counter.print_total_flops(show=True)
```

And this is output.
```
❯ python flopsTest.py
========== model ==========
/home/guest/anaconda3/envs/Compressor_3_10/lib/python3.10/site-packages/torch/overrides.py:110: UserWarning: 'has_cuda' is deprecated, please use 'torch.backends.cuda.is_built()'
  torch.has_cuda,
/home/guest/anaconda3/envs/Compressor_3_10/lib/python3.10/site-packages/torch/overrides.py:111: UserWarning: 'has_cudnn' is deprecated, please use 'torch.backends.cudnn.is_available()'
  torch.has_cudnn,
/home/guest/anaconda3/envs/Compressor_3_10/lib/python3.10/site-packages/torch/overrides.py:117: UserWarning: 'has_mps' is deprecated, please use 'torch.backends.mps.is_built()'
  torch.has_mps,
/home/guest/anaconda3/envs/Compressor_3_10/lib/python3.10/site-packages/torch/overrides.py:118: UserWarning: 'has_mkldnn' is deprecated, please use 'torch.backends.mkldnn.is_available()'
  torch.has_mkldnn,
total_flops =   0 (1 operations are ignored or not recognized)
```
My environment:
```
❯ conda list pytorch
# packages in environment at /home/guest/anaconda3/envs/Compressor_3_10:
#
# Name                    Version                   Build  Channel
pytorch                   2.1.0           py3.10_cuda11.8_cudnn8.7.0_0    pytorch
pytorch-cuda              11.8                 h7e8668a_5    pytorch
pytorch-mutex             1.0                        cuda    pytorch
```

**Comment by zugexiaodui** [comment]
The reason is that directly wrapping `nn.Linear` to a graph module will use the **function** of `linear` (_i.e._ `torch.nn.functional.linear`) rather than the **module** of `Linear` (_i.e._ `nn.Linear`). The function `linear` was not considered in the old version of `torch_flops`. This issue can be addressed in two ways:
+ Change from: `model = nn.Linear(dim, dim*3)` to: `model = nn.Sequential(nn.Linear(dim, dim*3))`
+ I have updated the operations in `torch_flops`. You can update the library to version 0.1.3 by running the following command: `pip install torch_flops --upgrade -i https://pypi.org/simple`. After updating, please try running your script again. I have tested your code, and it runs successfully.

**Comment by zugexiaodui** [comment]
Just ignore the warnings. They do not matter.

**Comment by BitCalSaul** [comment]
Thanks, I will try it later. Thanks for your hard working.

**Comment by BitCalSaul** [comment]
It works, thanks.
